ci/lint.sh

```diff
@@ -3,6 +3,9 @@
 # SPDX-FileCopyrightText: Red Hat, Inc.
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+# Activate the tests venv
+source /venv/bin/activate
```

Comment:
Where is that venv created?

Reply:
Here:
https://github.com/oVirt/vdsm/blob/master/docker/Dockerfile.centos-9#L77

But it is actually passing with the modification from this PR (which should be merged first):
https://github.com/oVirt/vdsm/pull/367

Includes the system packages in the venv.

Reply:
Thanks, I missed. Still, I think these shell scripts are (or at least were) useful outside of the container envs. How about we use `[ -d /venv ] && source /venv/bin/activate` with a comment that this is for virtual envs in container images?

Reply:
Fair point. I'll add that.

These scripts are in the `ci` folder and they finally rely on the `make` target, so outside the CIs they do not necessarily need to be used. But if they "could" be used before, I think we should keep that in the new approach.